### ISSUE TITLE
VideoClicks child-node attributes are missing

### ISSUE THREAD

**Issue by Grafluxe** [dailymotion/vast-client-js#131]
The ad object returned does not include the expected VideoClicks child-node attributes. Note, I'm testing for the ClickThrough id attribute.

```
<VideoClicks>
  <ClickThrough id="my-id">...</ClickThrough>
</VideoClicks>
```

See [IAB spec](https://www.iab.com/wp-content/uploads/2015/06/VASTv3_0.pdf), page 67

**Comment by michaelvial** [comment]
I don't think the parser can return the `<ClickThrough>` id attributes in the current architecture. Right now, only the URLs are returned. If we want to get there id attributes as well, we'll need to change `videoClickTrackingURLTemplates` structure to something like:

``` javascript
response.ads[0].creatives[0].videoClickTrackingURLTemplates = [
  {
    url : 'http://example.com/linear-clicktracking1',
    id : 'id1'
  },
  {
    url : 'http://example.com/linear-clicktracking2',
    id : 'id2'
  }
];
```
And this is an important breaking change. If we're going to do it, it'll be in the future v2. By curiosity, why do you need the id attribute for?

**Comment by Grafluxe** [comment]
It makes perfect sense to be in a v2 release. Thanks.

The reason we need access to the id attribute, is because we want our logic to fully support the IAB spec. For now we have written some override logic to handle it.

**Comment by kobawan** [comment]
Issue has been fixed in #324
